src/app/functions/server/onerep.ts

```diff
@@ -72,26 +55,18 @@ export async function createProfile(profileData) {
       `Failed to create OneRep profile: [${response.status}] [${response.statusText}]`
     );
   }
-  /**
-   * See https://docs.onerep.com/#operation/createProfile
-   *
-   * @type {{
-   *   id: number,
-   *   status: 'active' | 'inactive',
-   *   created_at: import('../utils/parse.js').ISO8601DateString,
-   *   updated_at: import('../utils/parse.js').ISO8601DateString,
-   *   url: string,
-   * }}
-   */
-  const savedProfile = await response.json();
+
+  const savedProfile: {
+    id: number;
+    status: "active" | "inactive";
+    created_at: import("../../../utils/parse.js").ISO8601DateString;
+    updated_at: import("../../../utils/parse.js").ISO8601DateString;
```

Comment:
Re: Since we're already importing from `../../../utils/parse.js` above in https://github.com/mozilla/blurts-server/blob/4eee0ae6bfbfcf0c5474f8f144127c1fbb3c52be/src/app/functions/server/onerep.ts#L7-L10

Can we import the `ISO8601DateString` type above on L9.5 or something and then just use the type here instead of dynamic type imports?:

```suggestion
    created_at: ISO8601DateString;
    updated_at: ISO8601DateString;
```